### PR TITLE
[INLONG-2229] Support pulsar anthentication in Manager and Sort

### LINKED ISSUE #2229: [Improve] Manager support pulsar authentification
### Description

_No response_

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kipshi**
### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #2229 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-service/src/test/resources/application-test.properties [hunk 1]
```diff
@@ -48,24 +42,34 @@ spring.datasource.druid.testOnReturn=false
...
 # Application name in Sort
 sort.appName=inlong_app
-
 # Pulsar admin URL
 pulsar.adminUrl=http://127.0.0.1:8080,127.0.0.2:8080,127.0.0.3:8080
 # Pulsar broker address
 pulsar.serviceUrl=pulsar://127.0.0.1:6650,127.0.0.1:6650,127.0.0.1:6650
 # Default tenant of Pulsar
 pulsar.defaultTenant=public
+# Audit configuration
```
Comment (dockerzhang) [inline inlong-manager/manager-service/src/test/resources/application-test.properties:62]: audit configuration is useless for this PR？

Comment (kipshi) [reply inlong-manager/manager-service/src/test/resources/application-test.properties:62]: audit configuration is default value, we use businessExtInfo to issue pulsar configuration. is user pulsar configuration is not issued, then use audit configuration instead;

Comment (kipshi) [reply inlong-manager/manager-service/src/test/resources/application-test.properties:62]: I think authentification should be issued by user in businessExtInfo, audit configuration can be used as default value, same  as pulsar url

**Review by baomingyu — APPROVED** [review]
+1

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**